### ISSUE TITLE
what's wrong  with the dictionary？

### ISSUE THREAD

**Issue by lijunforNLPtraining** [kamalkraj/BERT-NER#25]
thanks for your  sharing， but after downlading your code, there are always some editing error as the following:
![image](https://user-images.githubusercontent.com/44630102/61768412-f89d8900-ae19-11e9-9d0b-31014ba4952b.png)
I use PyCharm as my IDE tool, can you help to check what's wrong?  many thanks

**Comment by OthmaneJ** [comment]
It's supposed to be a dictionnary ('{' instead of '[') : 
output = {word:{"tag":label,"confidence":confidence} for word,label,confidence in zip(words,labels,logits_confidence)}

**Comment by kamalkraj** [comment]
ec7f643c679649bbd9e71ccde0528dd3e8ef5757
fixed
